tsdb/head.go

```diff
@@ -155,6 +156,15 @@ type HeadOptions struct {
 	EnableMemorySnapshotOnShutdown bool
 
 	IsolationDisabled bool
+
+	// Maximum number of CPUs that can simultaneously processes WAL replay.
+	// The default value is GOMAXPROCS.
+	// If it is set to a negative value or zero, the default value is used.
+	WALReplayConcurrency int
+}
+
+func getDefaultWALReplayConcurrency() int {
```

Comment:
Let's make this one a `var` assuming `const` is not possible. `DefaultWALReplayConcurrency`

Reply:
In fact, I was thinking about the same thing :)